pkg/api.go

```diff
@@ -215,7 +215,7 @@ func CheckStatusFromResult(result CheckResult) CheckStatus {
 	cs := CheckStatus{
 		Status:  result.Pass,
 		Invalid: result.Invalid,
-		Time:    time.Now().UTC().Format(time.RFC3339),
+		Time:    time.Now().UTC().Format(time.RFC3339Nano),
```

Comment:
This was required to return multiple results, previously the accuracy was upto seconds, and due to the unique(check_id, time) clause, only one incident was being added

Reply:
We shouldn't need this as each incident should be a different check_id ?